ui/chapters/src/main/java/de/danoeh/antennapod/ui/chapters/PodcastIndexTranscriptUtils.java

```diff
@@ -2,32 +2,44 @@
 
 import android.util.Log;
 
+import de.danoeh.antennapod.parser.transcript.TranscriptParser;
+import org.apache.commons.io.FileUtils;
+
 import java.io.File;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.nio.charset.Charset;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
 
 import de.danoeh.antennapod.model.feed.FeedMedia;
 import de.danoeh.antennapod.net.common.AntennapodHttpClient;
+import de.danoeh.antennapod.model.feed.Transcript;
+import okhttp3.CacheControl;
 import okhttp3.Request;
 import okhttp3.Response;
 import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.StringUtils;
 
 public class PodcastIndexTranscriptUtils {
 
     private static final String TAG = "PodcastIndexTranscript";
 
+
     public static String loadTranscriptFromUrl(String type, String url, boolean forceRefresh) {
         StringBuilder str = new StringBuilder();
         Response response = null;
+
         try {
             Log.d(TAG, "Downloading transcript URL " + url.toString());
-            Request request = new Request.Builder().url(url).build();
+            Request request = new Request.Builder().url(url).cacheControl(CacheControl.FORCE_NETWORK).build();
             response = AntennapodHttpClient.getHttpClient().newCall(request).execute();
             if (response.isSuccessful() && response.body() != null) {
                 str.append(response.body().string());
             }
-        } catch (IOException e) {
+        } catch (Exception e) {
```

Comment:
I replaced this because it prevented downloading files when the transcript code failed (in my case, because the transcript url was invalid)